### PR TITLE
Fix machine migration

### PR DESCRIPTION
**Pull request by mstanleyjones**
### Describe the proposed changes

Re-imported machine docs to get the 79 commits that were not part of the tag we imported. 

### Project version

n/a

### Related issue

#229

### Related issue or PR in another project

n/a

### Please take a look

@johndmulhausen @londoncalling @twang2218

### CODE REVIEW

**Comment by nathanleclaire** [comment]
Chatting a bit with @londoncalling about which of these seem fit for `master` and which for `vnext-machine`.  It seems that the driver-specific commits (virtualbox, amazon, azure are in there) are 0.9.0 features, so they should land in `vnext-machine`.  The rest look OK to land in master and deploy pretty much right away.

**Comment by nathanleclaire** [comment]
Looks like this might be missing https://github.com/docker/machine/commit/703f6d9c7 which @twang2218 had mentioned?

**Comment by johndmulhausen** [comment]
LGTM from a docs perspective

**Comment by twang2218** [comment]
LGTM, 3 commits I mentioned are back, as well as many other commits, thanks.

**Comment by mstanleyjones** [comment]
OK, I'm going to try to merge this as-is into `vnext-machine`, creating a new PR. Then I'll yank the version-specific commits out of this one and ping both for review again. Sound like a plan?

By the way, the SHAs will have changed due to some rebasing that was necessary, so you may need to match up the commit messages. I think they are all there.

**Comment by mstanleyjones** [comment]
@nathanleclaire PTAL again. I think I pulled out all the driver-specific commits that actually make doc changes. Let me know if I have made the right choices. In some cases I left commits in when they didn't actually change .md files, because the files they affected were removed during the actual import (for instance, `.go` files). When I removed some of these commits I had conflicts and had to make decisions, so I would appreciate your review of the result.

File: machine/drivers/azure.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ After authenticating, the driver will remember your credentials up to two weeks.
 
 [aad-docs]: https://azure.microsoft.com/documentation/articles/virtual-machines-windows-create-aad-work-id/
 
+    $ docker-machine create --driver azure --azure-subscription-id <subs-id> \
```
Comment (nathanleclaire) [inline machine/drivers/azure.md]: This seems off.  The `--azure-client-*` flags don't exist on Machine 0.8.2

**Comment by nathanleclaire** [comment]
Only one change seems off, rest seems good

File: machine/drivers/exoscale.md [hunk 2]
```diff
@@ -34,7 +36,7 @@ If a custom security group is provided, you need to ensure that you allow TCP po
 
 There is a limit to the number of docker machines that an anti-affinity group can have.  This can be worked around by specifying an additional anti-affinity group using `--exoscale-affinity-group=docker-machineX`
 
-Environment variables and default values:
+#### Environment variables and default values
```
Comment (mstanleyjones) [inline machine/drivers/exoscale.md]: Oops, this leading is off by one.

Comment (mstanleyjones) [inline machine/drivers/exoscale.md]: Fixed.

**Comment by nathanleclaire** [comment]
LGTM

**Comment by mstanleyjones** [comment]
@londoncalling can you please LGTM and merge?

**Pull request merged**